### ISSUE TITLE
Sorry to burst the bubble but this is Fake and will probably be a Scam in the later stage.

### ISSUE THREAD

**Issue by ShadyDev** [HumanAIGC/AnimateAnyone#15]
On X (formely twitter) people have found out the origin sources of the videos that open pose model is beeing generates from 

This tech is faked and be cautious on whatever this team is going to upload here

https://x.com/julesterpak/status/1731452110686892332?s=46&t=x02nRdMOO9e-APPWlVEkmA

**Comment by Jongulo** [comment]
I wish people that didn't understand how technology worked wouldn't make github accounts.

**Comment by ShadyDev** [comment]
If you really believe that the OG videos of the influencers weren't tempered on to seem like "AI Generated" than you're delusional. 
The only truly believable video is the one with Leo Messi but still that one can be also pretty much done with some simple Face Swapping. 

As the post on Twitter states - IF they want to prove they're legit they need to post a video of a one of those persons dancing with a completely different move set than the one it was "trained on (which I even doubt)".

**Comment by FurkanGozukara** [comment]
> If you really believe that the OG videos of the influencers weren't tempered on to seem like "AI Generated" than you're delusional. The only truly believable video is the one with Leo Messi but still that one can be also pretty much done with some simple Face Swapping.
> 
> As the post on Twitter states - IF they want to prove they're legit they need to post a video of a one of those persons dancing with a completely different move set than the one it was "trained on (which I even doubt)".

I also listed my red flags first day

**Comment by ricardoreis** [comment]
This is really strange. Let's wait for the source code to be published.

https://github.com/HumanAIGC/AnimateAnyone/assets/104947/046f981e-a25a-42ce-99d8-ee6c6598b5cd

**Comment by ShawnFumo** [comment]
Alright, I did a deep dive on this, and I believe it is NOT fake, though there is a lot of misunderstanding due to none of us being hooked into the research community.

First, the full site is here which has links to additional videos, the paper, etc:
https://humanaigc.github.io/animate-anyone/ 

In the paper they talk about doing three different training rounds. One which is more general and has a lot of different videos in it. This is what is being shown in most examples like Iron Man, 3d lady, lady with a hat and necklace, etc. You can see various moves applied to various characters, and many (all?) of those were never a video to start with.

They also trained separately on a specific TikTok dataset (which has been in various papers since 2021 it seems, first for making depth maps). This was specifically to test themselves against DisCo (more later). In this, they just trained on 300 TikTok videos from that dataset, and did 10 tests (from out of 100 test videos). For each of those, they ran it through pose detection and took a frame. Then they produced a video and compared it to how closely it matched the original TikTok test video, giving it numeric scores. But why recreate an existing video?

Because DisCo (from MS Azure and others) used this dataset and ran those tests on it. So this was an automated way they could say "our model is better than DisCo", with actual benchmark numbers. The DisCo site is here: https://disco-dance.github.io/    If you scroll down that page, you can see various cases where they apply diff animations to the same person or the same animation to diff people. It isn't AS good as AA, but still pretty good, and the code and models for that are linked.

They also did a third training (which I didn't look into as closely), on a particular fashion dataset (first used in 2019), because that was used in DreamPose, yet another project like this, that used that same dataset and ran benchmark numbers. You can see that here: https://grail.cs.washington.edu/projects/dreampose/   Not that DreamPose doesn't handle patterns on the clothes as well, and you can see in AA's videos that theirs does, which is why they do that comparison.

Also note that DisCo compared itself to DreamPose in their results. DreamPose references being better than yet two other papers from 2022 and 2021. And DisCo and DreamPose were both from earlier this year. All of these are doing similar things, with similar boring presentations, but it just happens that AA blew up on social media.

If you look at all of these in context, you can see the tech has been in the works for a while in diff forms, getting better with each iteration. Each time a paper and code is released, with sample videos, comparisons to the older projects, etc. I think people are judging these guys against companies like Runway and Pika Labs releasing advertising videos of new features. This seems more like pure research that maybe they didn't even expect people to notice that much (since DisCo and DreamPose don't seem to have gotten much press).

And to go back to the AA TikTok videos, my guess is that many of the videos in the training set had lipsyncing to songs, and may have even had multiple copies of the same meme dances with diff people as the ones in the test set. In that sense, it may be able to handle the motion of the highspeed dancing a bit better than the general case, but again it seems like that part was just them comparing how well it did on that task as compared to DisCo, while their main results were of animating diff still images with various pose animations, the exact thing people are saying they should show examples of. And those examples are still really good.

**Comment by Tybost** [comment]
_It sure is a nice day to look at the Stable Diffusion subreddit today._

**Comment by 5l1v3r1** [comment]
They have academic background to prove it is not fake, do you have anything to prove your claim or are you another desposable account just to post that kind of irrelevant comments?

**Comment by lucasjinreal** [comment]
It might not fake, but it can be overfitting on some real world existed video....

**Comment by xizaoqu** [comment]
> On X (formely twitter) people have found out the origin sources of the videos that open pose model is beeing generates from
> 
> This tech is faked and be cautious on whatever this team is going to upload here
> 
> https://x.com/julesterpak/status/1731452110686892332?s=46&t=x02nRdMOO9e-APPWlVEkmA

Actually, they have mentioned it in the paper Fig.5. It is a normal way to prove their effectiveness. You should read the paper first before making such idiot remarks.

**Comment by stacksize** [comment]
"I don't have access to the model, so it MUST BE FAKE"

**Comment by Collin-Budrick** [comment]
> Github used to be a peaceful, humble corner of mature, professional, polite folks on the internet until the flocks of normies jumping on the AI wave took over and turned it into a cesspool

Sad to say how true that is

**Comment by geekyayush** [comment]
> Github used to be a peaceful, humble corner of mature, professional, polite folks on the internet until the flocks of normies jumping on the AI wave took over and turned it into a cesspool

so true, it's being the new reddit
